README.md

```diff
@@ -94,4 +80,4 @@ Contributors:
 Original source code created by Stanford University is copyrighted under Apache 2 license and
 documented with an Apache 2.0 license header.
 
-The Profiled Editor was originally a work of the United States government and portions of project are in the public domain through the CC0 1.0 Universal public domain dedication license. 
+The Profiled Editor was originally a work of the United States government and portions of project are in the public domain through the CC0 1.0 Universal public domain dedication license.
```

Comment:
Typo? `s/Profiled/Profile/`

Reply:
probably took the typo right from lcnetdev.  Will fix in follow on PR (I promise)